docs/compass/03-02-labels.md

```diff
@@ -0,0 +1,158 @@
+  schema:{
+              "type": "array",
+              "items": {
+                  "type": "string",
+                  "enum": ["Go", "Java", "C#"]
+              }
+          }
+```
+
+If you want to add a new language to the list, provide such a mutation:
+
+```
+updateLabelDefinition(in: {
+                        key:"supportedLanguages",
+                        schema:{
+                                    "type": "array",
+                                    "items": {
+                                        "type": "string",
+                                        "enum": ["Go", "Java", "C#","ABAP"]
+                                    }
+                                }
+                      }) {...}
+```
+
+### Remove LabelDefinitions
+
+Use this mutation to remove a LabelDefinition:
+
+```graphql
+deleteLabelDefinition(key: String!, deleteRelatedLabels: Boolean=false): LabelDefinition
+
+```
+This mutation allows you to remove only definitions that are not used. If you want to delete a LabelDefinition with all values, set the **deleteRelatedLabels** parameter to `true`.
+
+## LabelFilters
+
+You can define a LabelFilter to list the top-level entities according to their labels. You can search for Applications and Runtimes by label keys or by label keys and their values. To search for a given Application or Runtime, use this query:
+
+```graphql
+ applications(filter: [LabelFilter!], first: Int = 100, after: PageCursor):  ApplicationPage!
+```
+
+To search for all objects with a given label despite their values, use:
+
+```graphql
+query {
+  applications(filter:[{key:"scenarios"
+    }]) {
+    data {
+      name
+      labels
+    }
+    totalCount
+  }
+}
+```
+
+To filter objects by their key and string value, use this query:
+
+```graphql
+runtimes(filter: { key: "{KEY}" query: "\"{VALUE}\"" })
+```
+
+You can also search for objects by their key and array values. In the **query** field, use only the limited SQL/JSON path expressions. The supported syntax is `$[*] ? (@ == "{VALUE}" )`. For example, to filter all objects assigned to the `default` scenario, run:
+
+```graphql
+query {
+  applications(filter:[{key:"scenarios",
+    query:"$[*] ? (@ == \"DEFAULT\")"}]) {
+    data {
+      name
+      labels
+    }
+    totalCount
+  }
+}
+```
+
+## **Scenarios** label
+
+Every Application is labeled with the special **Scenarios** label which by default has the `default` value assigned. As every Application has to be assigned to at least one scenario, if not specified explicitly, the `default` scenario is used.
+
+When you create a new tenant, the **Scenarios** LabelDefinition is created. It defines a list of possible values that can be used for the **Scenarios** label. Every time you create or modify an Application, there is a step that ensures that **Scenarios** label exists. You can add or remove values from the **Scenarios** LabelDefinition list, but neither the `default` value, nor the **Scenarios** label can be removed.
```

Comment:
If you add more scenarios as values, does the `default` value remain, or do they override it?

Reply:
It remains.